src/csharp/Grpc.nuspec

```diff
@@ -3,18 +3,17 @@
   <metadata>
```

Comment:
Do we really need to update Grpc.nuspec   in this PR? It seems that we don't. I'll be happy to accept cleanup of Grpc.nuspec as a separate PR (and please revert the file in this PR).

Reply:
Undone.